### PR TITLE
HomeVC Created

### PR DESCRIPTION
**Pull request by AJ-iOSDev**
First PR for HomeVC

### CODE REVIEW

File: AppOne/AppOne/SceneDelegate.swift [hunk 1]
```diff
@@ -17,6 +17,9 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
         // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
         guard let _ = (scene as? UIWindowScene) else { return }
+        window = UIWindow()
```
Comment (JoeHoHH) [inline AppOne/AppOne/SceneDelegate.swift:20]: actually you should give the `windowScene` here to the `window`, over it"s initializer

File: AppOne/AppOne/HomeVC.swift [hunk 2]
```diff
@@ -0,0 +1,15 @@
+//
+//  HomeVC.swift
+//  AppOne
+//
+//  Created by Asapu, Ajaikumar on 19.11.22.
+//
+
+import UIKit
+
+class HomeVC: UIViewController {
```
Comment (JoeHoHH) [inline AppOne/AppOne/HomeVC.swift:10]: class should be final if we dont plan to subclass

File: AppOne/AppOne/SceneDelegate.swift (hunk 1 above)
Comment (AJ-iOSDev) [reply AppOne/AppOne/SceneDelegate.swift]: @JoeHoHH Updated [here](https://github.com/JoeHoHH/ajayTheLead/pull/1/commits/82b81a4276343fca98befc7014edae74c8e77fa8)

File: AppOne/AppOne/HomeVC.swift (hunk 2 above)
Comment (AJ-iOSDev) [reply AppOne/AppOne/HomeVC.swift]: @JoeHoHH made it to final [here](https://github.com/JoeHoHH/ajayTheLead/pull/1/commits/82b81a4276343fca98befc7014edae74c8e77fa8)

File: AppOne/AppOne/SceneDelegate.swift [hunk 3]
```diff
@@ -16,8 +16,9 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
         // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
         // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
-        guard let _ = (scene as? UIWindowScene) else { return }
-        window = UIWindow()
+        guard let scene = (scene as? UIWindowScene) else { return }
+        window = UIWindow(frame: scene.coordinateSpace.bounds)
+        window?.windowScene = scene
```
Comment (JoeHoHH) [inline AppOne/AppOne/SceneDelegate.swift:21]: Actually there is a fitting init. `UIWindow(windowScene:)` So you can use this one and all is fine. Also i would duggest to get rid of all the empty generated `func's` we do not need atm

File: AppOne/AppOne/HomeVC.swift [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+//
+//  Created by Asapu, Ajaikumar on 19.11.22.
+//
+
+import UIKit
+
+final class HomeVC: UIViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .green
```
Comment (JoeHoHH) [inline AppOne/AppOne/HomeVC.swift:13]: for light and darkmode support we should use the .system colors instead of this.

Comment (JoeHoHH) [reply AppOne/AppOne/HomeVC.swift:13]: I know this is a placeholder thing, but for the future to remember. 💯

File: AppOne/AppOne/SceneDelegate.swift [hunk 5]
```diff
@@ -16,7 +16,11 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
         // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
         // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
-        guard let _ = (scene as? UIWindowScene) else { return }
+        guard let scene = (scene as? UIWindowScene) else { return }
```
Comment (JoeHoHH) [inline AppOne/AppOne/SceneDelegate.swift:16]: i also do not like the unneeded brackets, but this is just mz weirdness 😆

File: AppOne/AppOne/HomeVC.swift (hunk 4 above)
Comment (AJ-iOSDev) [reply AppOne/AppOne/HomeVC.swift]: Updated [here](https://github.com/JoeHoHH/ajayTheLead/pull/1/commits/a4e107c44a0e219e09b5d3c6165c235a278758de)

File: AppOne/AppOne/HomeVC.swift [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+//
+//  Created by Asapu, Ajaikumar on 19.11.22.
+//
+
+import UIKit
+
+final class HomeVC: UIViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .systemCyan
```
Comment (JoeHoHH) [inline AppOne/AppOne/HomeVC.swift:13]: 🥇

File: AppOne/AppOne/SceneDelegate.swift (hunk 3 above)
Comment (AJ-iOSDev) [reply AppOne/AppOne/SceneDelegate.swift]: updated [here]([url](https://github.com/JoeHoHH/ajayTheLead/pull/1/commits/a4e107c44a0e219e09b5d3c6165c235a278758de))

**Review by JoeHoHH — APPROVED** [review]

File: AppOne/AppOne/SceneDelegate.swift (hunk 5 above)
Comment (AJ-iOSDev) [reply AppOne/AppOne/SceneDelegate.swift:16]: how to get rid of braces here?

**Pull request merged**